share/completions/git.fish

```diff
@@ -798,6 +798,10 @@ format:\tSpecify which information to show"
     end
 end
 
+function __fish_git_is_rebasing
+    count (string join "/" (__fish_git rev-parse --show-toplevel --git-path rebase-))* > /dev/null
```

Comment:
The problem here is that `--git-path` will print absolute paths when inside a secondary worktree (`git worktree add`).
How about just using `--absolute-git-dir`? I don't think there are any relocation variables for the `rebase-merge` directory.

Reply:
Thanks for the suggestion! Applied in https://github.com/fish-shell/fish-shell/pull/8665/commits/9554cc8d64e8260de2e18b0bd434cf7ce38883e4.